subcommands/docker/cmd.go

```diff
@@ -29,6 +29,9 @@ func NewCommand() *cobra.Command {
 	}
 	helperPath = subcommands.FindWritableDirInPath(helperPath)
 	dockerConfigFile = dockerConfigPath()
+	if !subcommands.IsWritable(dockerConfigFile) {
+		dockerConfigFile = ""
+	}
```

Comment:
So, there are 3 modernizr commits; and then this one line of code which _fixes the thing_, right?
Can you extract it into a separate commit with a loud message like _it fixes xyz_?